### ISSUE TITLE
Vimium not working with some sites

### ISSUE THREAD

**Issue by drewauff** [philc/vimium#3929]
Vimium does not work properly with some sites, in my case Google and Github.

This is what happens on google. The same issues with Github and other sites but these are two of most frequented sites.

The vomnibar is a the grey part at the bottom and is full width of the window.

Just cant figure out why most sites work flawlessly and others ( very very few) dont.

![Screen Shot 2021-10-03 at 2 34 30 pm](https://user-images.githubusercontent.com/13850060/135738489-b1d85045-585d-40a4-a1de-f94c956cc44b.png)

This is that is supposed to look like:

![Screen Shot 2021-10-03 at 2 40 00 pm](https://user-images.githubusercontent.com/13850060/135738619-b8868c5b-f42e-4a81-ba60-5ef602420981.png)

 When I `o`, `ctrl-o`etc the vomnibar pops up it looks like the first screenshot until i start typing the it appears as it should.

**Comment by drewauff** [comment]
IMPORTANT NOTE: After a ton of testing and experimenting with different browsers, setttings etc. I have found it has something to do with cookies. If I delete all cookies Github and google work perfectly, once I sign in the problem is back.

**Comment by gdh1995** [comment]
Have you used some extensions to make them in dark mode?


How do you make Vomnibar dark? What's your customized CSS?



---Original---
From: ***@***.***&gt;
Date: Sun, Oct 3, 2021 12:31 PM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;
Subject: Re: [philc/vimium] Vimium not working with some sites (#3929)




 
IMPORTANT NOTE: After a ton of testing and experimenting with different browsers, setttings etc. I have found it has something to do with cookies. If I delete all cookies Github and google work perfectly, once I sign in the problem is back.
 
—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by drewauff** [comment]
Its the force dark mode chrome flag.  I have tried all modes, only one works even then I have to disable dark mode on the site which is not acceptable. I also use Dark Reader but websites look poop with that enabled.

**Comment by drewauff** [comment]
This is what is cause me grief
![Screen Shot 2021-10-03 at 7 16 17 pm](https://user-images.githubusercontent.com/13850060/135745714-1c06826e-7976-4b5f-ad07-ede54a8143c3.png)

None of them fix the issue, selected option is the closest. I have, however using DarkReader got Google to work but its far from ideal.

Dont know where to go from here........

**Comment by drewauff** [comment]
Ok, it is the "Force Dark Mode for Web Contents" that is causing the issue. When a website has its own "dark mode" the flag still forces its own dark mode.

Apparently something to do with `@media (prefers-color-scheme: dark)`.

If tf here was a way to "blacklist" certain sites from force dark mode that would fix everything..seems theres a lot people wanting the ability to `blacklist` or `whitelist` force dark mode. 

I have tried at least a dozen extensions, none work. The Devs, of Dark Reader say they may have found a way to plug into the "force dark mode"  but nothing more than that. 

So in a nutshell

1) Websites that have there own dark mode are broken due to force dark mode
2) There is no known way (at this point in time, at least that I could find) of fixing 1).

The search continues.....

**Comment by drewauff** [comment]
looks like im not the only one #3916 

IF there was a way to disable force dark mode on a per site basis that would be perfect.

**Comment by philc** [comment]
Closing due to age, and I'm not sure this is actionable for Vimium.
